packages/shared/components/MenuSshLogin/MenuSshLogin.tsx

```diff
@@ -100,7 +100,7 @@ export const LoginItemList = ({ logins, onClick, onKeyPress }) => {
         key={key}
         px="2"
         mx="2"
-        as={NavLink}
+        as={url ? NavLink : null}
```

Comment:
Could we change it so that instead of rendering a `div` when there's no URL it renders a `button` styled as a link instead? This actually provides keyboard support OOTB and helps us avoid having onClicks on divs.